tsnet/tsnet.go

```diff
@@ -178,8 +188,35 @@ func (s *Server) start() error {
 		return fmt.Errorf("%v is not a directory", s.rootPath)
 	}
 
-	// TODO(bradfitz): start logtail? don't use filch, perhaps?
-	// only upload plumbed Logf?
+	cfgPath := filepath.Join(s.rootPath, "tailscaled.log.conf")
```

Comment:
tsnet.log.conf?

Reply:
done

Reply:
oh, right 😕.  I don't have strong feelings on which it is, but yeah it should probably be consistent.

Reply:
The prior precedence occurs here: https://github.com/tailscale/tailscale/blob/09363064b544cd00e56fd4142e457119292e2588/tsnet/tsnet.go#L266